iree/compiler/Conversion/LinalgToSPIRV/KernelDispatchUtils.cpp

```diff
@@ -708,6 +752,27 @@ Optional<LaunchConfig> initGPULaunchConfig(
 #undef DISPATCH
   }
 
+  if (!rootOperation) {
+    for (linalg::LinalgOp linalgOp : linalgOps) {
+      if (auto op = dyn_cast<linalg::GenericOp>(linalgOp.getOperation())) {
+        if (getNumOuterParallelLoops(linalgOp) == 0 ||
+            llvm::any_of(linalgOp.getIndexingMaps(), [](AffineMap &map) {
```

Comment:
It might just be getting mixed with the old paths. This should not be needed actually.

Reply:
I still need this check otherwise some linalg op with weird affine map crash during tiling. I can look at why this happen after if you think this is not expected.

Reply:
Yeah, this is not expected. I dont see why it would carsh (at least would be good to know the affine map. Is this because of `reverse`? If so worth triaging upstream.

Reply:
Mahesh, yes the problem is with `reverse`, I'll work on fixing this part in a second step.